packages/dataprotector-consumer/prepare-legacy-dataprotector.js

```diff
@@ -0,0 +1,14 @@
+import * as fs from 'fs';
+
+const packageJsonPath = './node_modules/@iexec/dataprotector/package.json';
+
+console.log(`removing "exports" from ${packageJsonPath}`);
+
+const { exports, ...packageJson } = JSON.parse(
+  fs
+    .readFileSync(packageJsonPath, {
+      encoding: 'utf-8',
+    })
+    .toString()
+);
+fs.writeFileSync(packageJsonPath, JSON.stringify(packageJson));
```

Comment:
Not sure I understand what this file does :)

Maybe add some comments at top of the file?

Reply:
Yes, it strips the `exports` key from the `package.json` to allow us to import an otherwise not exported internal module.
I'll add this in comment.

Reply:
[2001c59](https://github.com/iExecBlockchainComputing/dataprotector-sdk/pull/151/commits/2001c59db05d5cbd3efb9f3b60ed8277b5de9f8a)